packages/commercetools/theme/nuxt.config.js

```diff
@@ -98,6 +98,9 @@ export default {
     endpoint: 'https://lovecrafts-dev.cdn.prismic.io/api/v2'
   },
   build: {
+    transpile: [
```

Comment:
Why we need that? Maybe add this to nuxt module (just do `if` checking if thats installed) same way as we have with sfui transpilation now

Reply:
https://logaretm.github.io/vee-validate/guide/rules.html#importing-rules-in-nuxt-js